src/Network/Network/PublicIpAddress/NewAzurePublicIpAddressCommand.cs

```diff
@@ -115,6 +115,12 @@ public class NewAzurePublicIpAddressCommand : PublicIpAddressBaseCmdlet
             HelpMessage = "The Domain Name label.")]
         public string DomainNameLabel { get; set; }
 
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "The Domain Name label Scope, the value can be TenantReuse or SubscriptionReuse or ResourceGroupReuse or NoReuse. It will decide HashedReusePolicy for FQDN, and can be set only when Domain Name Label has a valid value.")]
+        public PSDomainNameLabelScopeType DomainNameLabelScope { get; set; }
```

Comment:
> Looks like the DomainNameLabelScope has predefined values and cannot be arbitrarily. Have you tried to see if it would cause side effects when input string is not valid? I recommend using ValidateSet attribute to help with the validation if necessary.

added